lib/zclient.h

```diff
@@ -752,31 +771,33 @@ extern void redist_del_all_instances(struct redist_proto *red);
  * we have installed and play some special games
  * to get them both installed.
  */
-extern void zclient_send_vrf_label(struct zclient *zclient, vrf_id_t vrf_id,
-				   afi_t afi, mpls_label_t label,
-				   enum lsp_types_t ltype);
+extern enum zclient_send_status
+zclient_send_vrf_label(struct zclient *zclient, vrf_id_t vrf_id, afi_t afi,
+		       mpls_label_t label, enum lsp_types_t ltype);
 
 extern void zclient_send_reg_requests(struct zclient *, vrf_id_t);
 extern void zclient_send_dereg_requests(struct zclient *, vrf_id_t);
-extern int zclient_send_router_id_update(struct zclient *zclient,
-					 zebra_message_types_t type, afi_t afi,
-					 vrf_id_t vrf_id);
-
-extern void zclient_send_interface_radv_req(struct zclient *zclient,
-					    vrf_id_t vrf_id,
-					    struct interface *ifp, int enable,
-					    int ra_interval);
-extern int zclient_send_interface_protodown(struct zclient *zclient,
-					    vrf_id_t vrf_id,
-					    struct interface *ifp, bool down);
+extern enum zclient_send_status
+zclient_send_router_id_update(struct zclient *zclient,
+			      zebra_message_types_t type, afi_t afi,
+			      vrf_id_t vrf_id);
+
+extern enum zclient_send_status
+zclient_send_interface_radv_req(struct zclient *zclient, vrf_id_t vrf_id,
+				struct interface *ifp, int enable,
+				int ra_interval);
+extern enum zclient_send_status
```

Comment:
the caller of this in vrrp_zebra.c is just returning the enum as an int - doesn't look like that return is begin used, though

Reply:
yep and the return is ignored everywhere. Yoinking it.